docs/tasks/configure-pod-container/tcp-liveness.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: goproxy
+  labels:
+    app: goproxy
+spec:
+  containers:
+  - name: goproxy
+    image: gcr.io/google_containers/goproxy:0.1
+    ports:
+    - containerPort: 8080
+    readinessProbe:
```

Comment:
The name of this yaml says `liveness`, but it has a `readinessProbe`. I'd recommend adding multiple examples each showing all the available fields/features..